### PR TITLE
fix(react-card): add `react-components` to vr-tests asset path

### PR DESCRIPTION
**Pull request by andrefcdias**
## Current Behavior

Due to the moving of the package, the usage of hardcoded absolute URLs to access GitHub raw assets in our stories is using the wrong path.

## New Behavior

Path now accounts for the `react-components/` move.

## Related Issue(s)

#22692

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3e0ff753cbb3375f28477b2249ecd8b012e2e946'>3e0ff753cbb3375f28477b2249ecd8b012e2e946</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by andrefcdias** [comment]
Screener fail is expected, will approve the changes after I get PR approval.

**Review by ling1726 — APPROVED** [review]

**Pull request merged**
